UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
Ah! Interesting, and makes sense indeed!

But the proper fix would be to patch my scripts so that they can handle paths with spaces, so that it works for everyone :wink:  I'm guessing that would just be a matter of wrapping some `$XX` variables into quotes so should be easy to test at least.